lang/cs/Org.Apache.REEF.IO.Tests/TestAzureBlockBlobFileSystemE2E.cs

```diff
@@ -138,6 +139,43 @@ public void TestDeleteE2E()
             Assert.False(CheckBlobExists(blob));
         }
 
+        [Fact(Skip = SkipMessage)]
+        public void TestGetChildrenE2E()
+        {
+            string[] parentNames = new string[] { "sample1", "sample2", "sample3" };
+            string[] folderNames = new string[] { "folder1/sample4", "folder1/sample5" };
+            string[] allNames = parentNames.Concat(folderNames).ToArray();
+
+            foreach (string uploadedBlobName in allNames)
+            {
+                CloudBlockBlob blob = _container.GetBlockBlobReference(uploadedBlobName);
+                UploadFromString(blob, "hello");
+            }
+
+            // List all files in the container
+            CloudBlobContainer container = _client.GetContainerReference(_container.Name);
+            IEnumerable<Uri> blobs = _fileSystem.GetChildren(container.Uri).ToList();
+            IEnumerable<string> allBlobNames = blobs.Select(b => b.LocalPath.Replace("/" + _container.Name + "/", string.Empty));
+
+            foreach (string blobName in allBlobNames)
+            {
+                Assert.Contains(blobName, allNames);
+            }
+
+            Assert.Equal(allNames.Count(), blobs.Count());
+
+            // List files only in the sub-folder in the container
```

Comment:
The second sections of this test is identical to the first section of this test. Please consolidate to a helper method.

Reply:
Consolidated code into a common method.